scripts/run-dljc.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+WORKING_DIR=$(pwd)
+
+export MYDIR=`dirname $0`
+. ./$MYDIR/setup.sh
+
+JSR308=$JSR308
```

Comment:
This variable isn't used in other scripts and isn't set in setup.sh.
Can you please remove it and just do it relative to `MYDIR`, like it's done in other scripts?